editor/src/core/shared/element-template.ts

```diff
@@ -2533,6 +2533,7 @@ export interface ElementInstanceMetadata {
   conditionValue: ConditionValue
   textContent: string | null
   earlyReturn: EarlyReturn | null
+  prop: string | null
```

Comment:
I think prop is not a very helpful name, especially because a jsx element has props itself.
Can this be something more verbose and clear? E.g. containingRenderProp or something like that?

Reply:
Yeah as mentioned I know the name is bad. I'm going to go with `assignedToProp` (or `assignedToRenderProp` perhaps) and see how that feels

Reply:
Ok done - I went with `assignedToProp`. How do you feel about that?

Reply:
`assignedToProp` is fine by me!